src/styles/products.scss

```diff
@@ -0,0 +1,11 @@
+.products {
+  max-width: 944px;

```

Comment:
According to the task checklist, calculate it.
![Screenshot 2022-07-21 at 20 06 41](https://user-images.githubusercontent.com/17098736/180272672-c8b813be-87ea-458d-9b8a-f4efb8e04060.png)